spec/requests/users_requests_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns index page view as html' do
+      get '/'
+
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:index)
```

Comment:
- [ ] Great job working on the tests so far but I think it would be great if you add tests for `If the response body includes correct placeholder text.` , since I just could be able to see that you tested the two cases kindly test the third case as mentioned in the requirements 💯⭐🤩 


![image](https://github.com/simplegoose/blog-app/assets/58553711/1815f7b1-74f5-4d83-b047-f66b6c58f181)